processes/WIP-bug_bounty_proposal.md

```diff
@@ -0,0 +1,73 @@
+# Proposal to advance a Bug Bounty program for the Node.js® organization
+
+## Introduction
+
+Security is an essential part of the ongoing success of the Node.js community. The community is a vital part of one of the most aggressive markets available providing critical runtime infrastructure for a growing percentage of the internet and corporate computing requirements in 2018. A bug bounty program has become a standard mechanism to improve the security posture of projects, organizations and companies at every level.
+
+A bug bounty program is a mechanism to formally engage security researchers and developers in a way that provides benefits to the community along multiple vectors:
+
+1. Improved Security Depth of Coverage • Finding security defects is specialized work that benefits from deep expertise and fresh eyes
+1. Validate Nodejs as a secure platform • Expanding the community into the security world is a growth area and enables new audiences for nodejs to be proposed
+1. Improved Security Scope of Coverage • Reduced Risk we have bugs that are undiscovered currently exposing ourselves and our users to unnecessary risk
+1. Improved Process in SDLC • Ongoing analysis of discoveries provides an opportunity to improve security through practices and/or security focus and tooling.
+
+## Proposal Process
+
+The proposed process for considering a Bug Bounty program makes heavy reference to existing work that the Security WG has already started (establishing a triage team, organized disclosure system, etc.) and ongoing partnership with HackerOne. High level process:
+
+-[ ] Security WG reviews readiness and desire to have such a program (presume yes unless someone objects)
+-[ ] Security WG presents proposal to TSC (see next steps on how to get there)
+-[ ] TSC / Security WG bring funding proposal before Board
+-[ ] Security WG & TSC establish Bug Bounty Leader & Bug Bounty Team (will take additional coordination across org)
+
+## Next steps
+
+1. Core team readiness review and architectural decisions
+
+   - goals/kpi (scope, responsiveness goals, impact of defects, team size, budget)
+   - private vs. public
+   - existing security analysis
+   - backlog validation
+
+1. Engage other committies to let them know of the proposal (community committee as there is a PR element, others?)
+1. Review and evolve draft note into a proposal
+1. Create draft Security Page updates that describe the rules of the road
+
+## Readiness review and questions for input
+
+As preparation, the Security WG already has an organization within HackerOne and the general framework as well as the technical tools to offer a Bug Bounty program.
+
+1. Do we have a security assessment of the current node.js core? *no formal assessment, should not block, but increases risk of public bounty program*
+1. Do we have any scope limitations on what we would declare? *yes, we do, just what are they exactly?*
```

Comment:
Definitely https://github.com/nodejs/node at the minimum.
What other projects (if any) would make sense?